lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -119,6 +120,9 @@ def get(self, request, *args, **kwargs):
         show_handouts = show_enrolled or allow_public
         handouts_html = get_course_info_section(request, request.user, course, 'handouts') if show_handouts else ''
 
+        # TODO: TNL-7185 Legacy: Refactor to return the offer data and format the message in the MFE
+        offer_html = generate_offer_html(request.user, course_overview)
```

Comment:
This is copied from TNL's metadata endpoint, where they also send along offer_html.